tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py26,py27,pypy,py34,py35,py36,py37
+envlist = python
```

Comment:
Tox

Reply:
ok, let me explain: here it was modified so by default it does not run against all supported versions, rendering this config next to useless

Default should still be to run against all supported versions so every contributor can check hers/his code before making a PR and triggering external CI.

Reply:
`tox -e py26,py27,pypy,py34,py35,py36,py37`

Reply:
Most of the time you want to test against the current version of Python you have in your env. Everything else will be caught in CI. When something breaks you'll select certain versions to test against because it'd take too much time to run everything on minor changes. It's bad dev UX.

Reply:
I would say its other way around. If I wanted to test just a single "default" env, then why would I use tox in the first place?

Maybe it would be good idea to move this change to other PR instead sneaking it in here?